otter/auth.py

```diff
@@ -284,6 +284,42 @@ def __hash__(self):
                      self._url, self._admin_url))
 
 
+@implementer(IAuthenticator)
+class SingleTenantAuthenticator(object):
+    """
+    An authentication handler that authenticates using a single tenant id.
+    """
+    def __init__(self, _identity_user, _identity_password, url):
```

Comment:
Damnit, I was typing that up :-)

Also, "not actually Keystone" is a reasonable constraint. Up until very recently (and still, at present?) this was the case at a little hosting company called Rackspace ;)